Coding style error in marine convert scripts

@gthompsnJCSDA got the following error when running ctest:

```
3/29 Testing: iodaconv_marine_coding_norms

3/29 Test: iodaconv_marine_coding_norms

Command: "/data/users/gthompsn/JEDI/build/build_ioda-converters/bin/iodaconv_lint.sh" "/data/users/gthompsn/JEDI/src/ioda-converters/src/marine" "/data/users/gthompsn/JEDI/src/ioda-converters"

Directory: /data/users/gthompsn/JEDI/build/build_ioda-converters/src/marine

"iodaconv_marine_coding_norms" start time: Jun 03 18:20 UTC

Output:

----------------------------------------------------------

+ SRC_DIR=/data/users/gthompsn/JEDI/src/ioda-converters/src/marine

+ PYCODESTYLE_CFG_PATH=/data/users/gthompsn/JEDI/src/ioda-converters

+ cd /data/users/gthompsn/JEDI/src/ioda-converters/src/marine

+ pycodestyle -v --config=/data/users/gthompsn/JEDI/src/ioda-converters/.pycodestyle '--filename=*.py,*.py.in' .

cli configuration: /data/users/gthompsn/JEDI/src/ioda-converters/.pycodestyle

directory .

checking ./argoClim2ioda.py.in

checking ./cryosat_ice2ioda.py.in

checking ./cryosat_ice2ioda_DBL.py.in

checking ./emc_ice2ioda.py.in

checking ./gds2_sst2ioda.py.in

checking ./gmao_obs2ioda.py.in

checking ./godae_profile2ioda.py.in

./godae_profile2ioda.py.in:198:21: E741 ambiguous variable name 'l'

checking ./godae_ship2ioda.py.in

checking ./godae_trak2ioda.py.in

checking ./hgodas_adt2ioda.py.in

checking ./hgodas_insitu2ioda.py.in

checking ./hgodas_sst2ioda.py.in

checking ./rads_adt2ioda.py.in

checking ./smap_sss2ioda.py.in

checking ./viirs_modis_oc2ioda.py.in

1       E741 ambiguous variable name 'l'

<end of output>

Test time =   0.81 sec

----------------------------------------------------------

Test Failed.

"iodaconv_marine_coding_norms" end time: Jun 03 18:20 UTC

"iodaconv_marine_coding_norms" time elapsed: 00:00:00

----------------------------------------------------------
```

It seems that this has surfaced due to a newer version of pycodestyle. The ``iodaconv_marine_coding_norms`` test passes on my iMac. I've got version 2.5.0 of pycodestyle installed, and @gthompsnJCSDA is using version 2.6.0.

--- a/src/marine/godae_profile2ioda.py.in
+++ b/src/marine/godae_profile2ioda.py.in
@@ -195,14 +195,14 @@ class IODA(object):
                 lon = obs.data['ob_lon'][n]
                 dtg = datetime.strptime(obs.data['ob_dtg'][n], '%Y%m%d%H%M')
 
-                for l, lvl in enumerate(obs.data['ob_lvl'][n]):
+                for ilev, lvl in enumerate(obs.data['ob_lvl'][n]):
 
                     locKey = lat, lon, lvl, dtg.strftime("%Y-%m-%dT%H:%M:%SZ")
 
                     for key in self.varDict.keys():
 
-                        val = obs.data[key][n][l]
-                        err = obs.data[key+'_err'][n][l]
+                        val = obs.data[key][n][ilev]
+                        err = obs.data[key+'_err'][n][ilev]
                         qc = (100 * obs.data[key+'_qc'][n]).astype('i4')
 
                         valKey = self.keyDict[key]['valKey']

